requirements.txt

```diff
@@ -1,16 +1,16 @@
 certifi==2022.12.7
-chardet==3.0.4
-idna==2.9
-numpy==1.23.4; python_version >= '3.8'
+chardet==5.1.0
```

Comment:
3.6 build fails

> ERROR: Could not find a version that satisfies the requirement chardet==5.1.0 (from versions: 1.0, 1.0.1, 1.1, 2.1.1, 2.2.1, 2.3.0, 3.0.0, 3.0.1, 3.0.2, 3.0.3, 3.0.4, 4.0.0, 5.0.0

Reply:
It's been a bit of a chore, but the checks do seem to be working well.
And the run-check.py being able to run locally (though I only have 1 version installed) is great 👍🏻